### ISSUE TITLE
AttributeError: 'OutStream' object has no attribute 'buffer'

### ISSUE THREAD

**Issue by PasaOpasen** [roedoejet/g2p#43]
```python
from g2p import make_g2p
```
```
Traceback (most recent call last):

  File "<ipython-input-1-7add92db992e>", line 1, in <module>
    from g2p import make_g2p

  File "C:\ProgramData\Anaconda3\lib\site-packages\g2p\__init__.py", line 18, in <module>
    sys.stdout = io.TextIOWrapper(sys.stdout.buffer, encoding="utf8")

AttributeError: 'OutStream' object has no attribute 'buffer'
```

Last version of g2p and python 3.7.7

**Comment by roedoejet** [comment]
It looks like you're using some sort of non-default stdout which doesn't support the buffer attribute. I've patched this (64b44b7), so that it should run, but if your stdout doesn't use utf8, then you might run into problems too.

Please clone the latest build (in master branch) and let me know if that works so I can close the issue.

**Comment by PasaOpasen** [comment]
How it's okay)
